app/presenters/all_content_workflow_presenter.rb

```diff
@@ -0,0 +1,39 @@
+class AllContentWorkflowPresenter < CSVPresenter
+  def initialize(scope = Edition.all)
+    super(scope)
+    self.column_headings = %i[
+      content_title
+      content_slug
+      content_url
+      current_status
+      stage
+      format
+      current_assignee
+      created_at
```

Comment:
Do we need to have `date_created` and `time_created` in the `initialize` function when we have `created_at` (assuming we keep all 3 in the `build_csv` method)?

Reply:
This was explicitly requested by those who are consuming the data - the _better_ thing to do in my mind would be to remove the `created_at` field, but I left it in there for consistency with the other reports 🤔

Reply:
Just to be _overbearingly_ explicit, the `column_headings` array in the `initialize` function need to match perfectly with what we create in the `build_csv` function (as this all needs to line up properly in the right order on the csv file). So, if we add or remove something from the first array then we need to do the same in the one below, yeah? 😄